packages/analytics-js-integrations/src/integrations/Sprig/browser.js

```diff
@@ -12,7 +12,7 @@ class Sprig {
             logger.setLogLevel(analytics.logLevel);
         }
         this.analytics = analytics;
-        this.apiKey = config.apiKey;
+        this.environmentId = config.environmentId;
```

Comment:
Won't this be a breaking change?

Reply:
yes, but this destination as released few days ago so no new connections are there

Reply:
Looks like this is a new destination with no new connections cc @mihir-4116